src/screens/Sensors/index.tsx

```diff
@@ -64,9 +61,17 @@ export default function Sensors({navigation}: DrawerNavProps) {
     // Remove the selected sensor and refetch data.
     try {
       await item.deleteSensor();
-      await ble.disconnect(item.address).catch((err: Error) => {
-        console.log(err);
-      });
+      if (item.sensorType.includes('HeartRate')) {
```

Comment:
We have tests that run through deleting a sensor, you could add an expect to ensure we call heartRateMonitor.disconnect.